pom.xml

```diff
@@ -0,0 +1,200 @@
+        should be bundled in a jar (or war, or whatever) and get specified here
+        -->
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>example.properties</include>
+                </includes>
+                <targetPath>gov/nasa/pds/mypackage</targetPath>
+            </resource>
+            <!-- You can give <testResources> for resources needed during the test phase -->
+        </resources>
+
+        <!--
+        Plugins tell Maven how to operate the various lifecycle phases; include extra as needed, such
+        as for CORBA stub generation from `.idl` files, or even more modern things.
+        -->
+        <plugins>
+            <!-- This plugin actually compiles the `.java` source code, so you probably need it! -->
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <useIncrementalCompilation>false</useIncrementalCompilation>
+                </configuration>
+            </plugin>
+            <!-- This plugin runs the testing framework, so you also probably need it -->
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.2</version>
+            </plugin>
+            <!--
+            This plugin makes this package an *executable* jar. If you want to make a plain jar, adjust
+            the parameters below; or use a different plugin to get a differnt kind of artifact.
+            -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                        </manifest>
+                        <manifestEntries>
+                            <Main-Class>gov.nasa.pds.mypackage.Main</Main-Class>
+                        </manifestEntries>
+                    </archive>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <!-- Other handy items get inherited from the parent POM, but you can override these here -->
+    <developers/>
+    <profiles/>
+    <distributionManagement/>
+    <reporting/>
+    <licenses/>
+    <!--
+    The <organization> should be inherited from the parent but isn't there yet, so we include it explicitly
+    here in this template `pom.xml`. Of course, you'll still be able to override it.
+    -->
+    <organization>
```

Comment:
@nutjob4life if we use pds parent, then i don't think we need this?

Reply:
Just don't forget to add `<organization>` to `gov:nasa:pds:1.9.0:pom` 👍 

Removed `<organization>` from here in 52b4149f5be360ad91b83281190013aa681a9e55